doc/UsersGuide/source/omjulia.rst

```diff
@@ -78,6 +78,11 @@ The object constructor requires a minimum of 2 input arguments which are strings
 
 >>> ModelicaSystem(mod,"BouncingBall.mo","BouncingBall",["Modelica", "SystemDynamics", "dcmotor.mo"])
 
+-  The fourth input argument (optional), is a keyword argument which is used to set the commandLineOptions e.g.,
```

Comment:
Change `commandLineOptions` to `command line options`

Reply:
updated